age/smart-contract/assembly/smart-contract.ts

```diff
@@ -0,0 +1,19 @@
+/* Tic Tac Toe Implementation for Massa Labs
+ *
+ * */
+import { Storage, generateEvent, Args, toBytes, fromBytes } from "@massalabs/massa-as-sdk";
```

Comment:
```suggestion
import { Storage, Args, toBytes } from "@massalabs/massa-as-sdk";
```
`fromBytes` and `generateEvent` are not used.